server/src/main/java/org/opensearch/cluster/coordination/Coordinator.java

```diff
@@ -761,7 +774,7 @@ void becomeFollower(String method, DiscoveryNode leaderNode) {
         lastKnownLeader = Optional.of(leaderNode);
         peerFinder.deactivate(leaderNode);
         clusterFormationFailureHelper.stop();
-        closePrevotingAndElectionScheduler();
+        closePrevoting();
```

Comment:
A node becomes follower as soon as it receives a publish request. If we close the election scheduler and if this publication does not succeed then the node will become candidate and restart election without any backoff.
We want to keep the election scheduler open until the cluster state has been applied after the election so that the election always starts with some backoff. Even though the election scheduler is open when the node is follower, the logic for election will only execute if a node is candidate.

https://github.com/opensearch-project/OpenSearch/blob/fbcc0d94d1183e15ea3c74c72894c527460b2ce3/server/src/main/java/org/opensearch/cluster/coordination/Coordinator.java#L1483-L1491

Reply:
when will close for a follower, which code path will handle it?